jest.config.js

```diff
@@ -1,65 +1,60 @@
+    '^.+\\.js$': 'babel-jest',
+    '^.+\\.jsx$': 'babel-jest',
+  },
+  displayName: 'Unit Tests',
+  collectCoverageFrom: [
+    '**/(src|scripts)/**/*.{js,jsx}',
+    '!**/src/testHelpers/**',
+    '!**/*.stories.jsx',
+    '!**/src/integration/!(utils)/**/*',
+  ],
+  testMatch: [
+    '**/__tests__/**/*.js?(x)',
+    '**/?(*.)+(spec|test).js?(x)',
+    '!**/src/integration/!(utils)/**/*',
+  ],
+};
+
+const ampIntegrationTests = {
+  displayName: 'Integration Tests - AMP',
+  testEnvironment: './src/integration/integrationTestEnvironment.js',
+  testEnvironmentOptions: {
+    platform: 'amp',
+  },
+  setupFilesAfterEnv: ['./src/testHelpers/setupTests.js'],
+  testMatch: ['**/src/integration/!(utils)/**/*[^.canonical].test.js'],
+};
+
+const canonicalIntegrationTests = {
+  displayName: 'Integration Tests - Canonical',
+  testEnvironment: './src/integration/integrationTestEnvironment.js',
+  testEnvironmentOptions: {
+    platform: 'canonical',
+  },
+  setupFilesAfterEnv: ['./src/testHelpers/setupTests.js'],
+  testMatch: ['**/src/integration/!(utils)/**/*[^.amp].test.js'],
+};
+
 module.exports = {
-  projects: [
-    {
-      displayName: 'Unit Tests',
-      reporters: [
-        'default',
-        [
-          'jest-junit',
-          {
-            suiteName: 'Unit Tests',
-            outputDirectory: 'reports/jest',
-            uniqueOutputName: 'true',
-            ancestorSeparator: ' › ',
-          },
-        ],
-      ],
-      collectCoverageFrom: [
-        '**/(src|scripts)/**/*.{js,jsx}',
-        '!**/src/testHelpers/**',
-        '!**/*.stories.jsx',
-        '!**/src/integration/**/*.{js,jsx}',
-      ],
-      setupFiles: ['./src/testHelpers/jest-setup.js'],
-      testMatch: [
-        '**/__tests__/**/*.js?(x)',
-        '**/?(*.)+(spec|test).js?(x)',
-        '!**/src/integration/**/*.{js,jsx}',
-      ],
-      snapshotSerializers: ['enzyme-to-json/serializer'],
-      setupFilesAfterEnv: ['./src/testHelpers/setupTests.js'],
-      moduleNameMapper: jestDirAlias,
-      transform: {
-        '^.+\\.js$': 'babel-jest',
-        '^.+\\.jsx$': 'babel-jest',
-      },
-    },
-    {
-      displayName: 'Integration Tests',
-      testEnvironment: './src/integration/integrationTestEnvironment.js',
-      reporters: [
-        'default',
-        [
-          'jest-junit',
-          {
-            suiteName: 'Integration Tests',
-            outputDirectory: 'reports/jest',
-            uniqueOutputName: 'true',
-            ancestorSeparator: ' › ',
-          },
-        ],
-      ],
-      setupFiles: ['./src/testHelpers/jest-setup.js'],
-      testMatch: ['**/src/integration/**/*.test.{js,jsx}'],
-      snapshotSerializers: ['jest-serializer-html'],
-      setupFilesAfterEnv: ['./src/testHelpers/setupTests.js'],
-      moduleNameMapper: jestDirAlias,
-      transform: {
-        '^.+\\.js$': 'babel-jest',
-        '^.+\\.jsx$': 'babel-jest',
+  projects: [unitTests, ampIntegrationTests, canonicalIntegrationTests],
```

Comment:
@andrewscfc We have already excluded integration tests from the unit test command (`test:unit`) in package.json https://github.com/bbc/simorgh/blob/ea216180db8ab8d2f17c8352879e78bae86c33f0/package.json#L34

Reply:
Okay, probably not understanding. If we are excluding integration tests in the command, why are we including them in this config: https://github.com/bbc/simorgh/pull/6168/files/04fda4d821dc7b03ffde9135cc7cc0213c326674#diff-2d0cd5d10b9604941c38c6aac608178aR22 

I'm sure I'm missing something, apologies

Reply:
That glob excludes the integration tests except for anything in the `utils` folder of integration tests because they contain unit tests. They don't test an integration, they literally test individual units of code.